backend/expense-calculator/src/main/resources/expence-calculator-service-persister.yml

```
@@ -0,0 +1,142 @@
+
+            - jsonPath: $.BillMetaCalculation.*.billMetaCalcDetails.*.additionalDetails
+              type: JSON
+              dbType: JSONB
+
+            - jsonPath: $.BillMetaCalculation.*.billMetaCalcDetails.*.auditDetails.createdBy
+
+            - jsonPath: $.BillMetaCalculation.*.billMetaCalcDetails.*.auditDetails.lastModifiedBy
+
+            - jsonPath: $.BillMetaCalculation.*.billMetaCalcDetails.*.auditDetails.createdTime
+
+            - jsonPath: $.BillMetaCalculation.*.billMetaCalcDetails.*.auditDetails.lastModifiedTime
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)